Keas.Core/Domain/History.cs

```diff
@@ -28,8 +28,13 @@ public History()
         // Created, granted, accepted, declined, revoked, deleted 
         public string ActionType { get; set; }
 
+        // TODO: Add serial? Or just replace key with serial????
```

Comment:
@jcubbage we probably want both anyway so remove this comment and i think the way you implemented it is a good plan